tokens/build-tokens.js

```diff
@@ -2,19 +2,22 @@
 const { program } = require('commander');
 const path = require('path');
 const { StyleDictionary, colorTransform, createCustomCSSVariables } = require('./style-dictionary');
+const { createIndexFile } = require('./utils');
 
 program
   .version('0.0.1')
   .description('CLI to build design tokens for various platforms (currently only CSS is supported) from Paragon Design Tokens.')
   .option('--build-dir <char>', 'A path to directory where to put files with built tokens, must end with a /.', './build/')
   .option('--source <char>', 'A path where to look for additional tokens that will get merged with Paragon ones, must be a path to root directory of the token files that contains "root" and "themes" subdirectories.')
   .option('--source-tokens-only', 'If provided, only tokens from --source will be included in the output; Paragon tokens will be used for references but not included in the output.')
+  .option('--themes <themes...>', 'A list of all the themes do you want to build, by default Paragn build light theme.')
```

Comment:
nit grammar and typo. Perhaps something like:

```suggestion
  .option('--themes <themes...>', 'A list of theme variants to build. By default, Paragon currently only supports a light theme.')
```